stateresolutionv2.go

```diff
@@ -583,3 +603,98 @@ resetNoIncoming:
 	//sort.Sort(sort.Reverse(stateResV2ConflictedPowerLevelHeap(graph)))
 	return graph
 }
+
+// kahnsAlgorithmByAuthEvents is, predictably, an implementation of Kahn's
+// algorithm that uses auth events to topologically sort the input list of
```

Comment:
Comments still mention "auth events here".

(There's a few instances in the comments in the function, too.)